config/database.yml

```
@@ -1,22 +1,22 @@
-  <<: *default
-  database: itsm_api_development
-
-test:
-  <<: *default
-  database: itsm_api_test
-
-production:
-  <<: *default
-  database: itsm_api_production
-  username: itsm_api
+default: &default
```

YAMLlint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)